test/Conversion/CalyxToHW/primitives.mlir

```diff
@@ -0,0 +1,44 @@
+// RUN: circt-opt --split-input-file -lower-calyx-to-hw --canonicalize %s | FileCheck %s
```

Comment:
Please don't invoke `--canonicalize`, we want to test `--lower-calyx-to-hw` in isolation.

Reply:
Gravitated towards a `--canonicalize` call since the output looks like the intention of the lowering, minus the wires. But you are right, the exact output should be tested - will fix.

Reply:
Yeah, it is annoying, but I think for the best.